pkg/prompts/prompts.go

```diff
@@ -127,6 +143,65 @@ func validateBiggerThanZero(input string) error {
 	return nil
 }
 
+func (r *realPrompter) CaptureListDecision(
+	prompter Prompter, // we need this in order to be able to run mock tests
+	prompt string,
+	capture func(prompt string, args any) (any, error),
+	capturePrompt string,
+	label string,
+	info string,
+	arg any,
```

Comment:
shall avoid the arg, and instead give a capture function with the param already included

Reply:
I am not sure in this design the `arg` can be avoided. It is _not_ representing the capture function, but an **optional** argument for the actual capture function. It can not be the capture function because the two capture functions supported do not share the same signature (`CapturePChainAddress` has an extra arg `network`, which is passed through this `arg` here).

Reply:
As I mentioned somewhere else, I have preference for avoiding this extra work of passing around this arg to the capture function, forcing some signatures.